### PR TITLE
Restructure activities/codebasestewardship

### PR DESCRIPTION
**Pull request by clausmullie**
Proposes a fix for #1076 

-----
[View rendered activities/codebase-stewardship/index.md](https://github.com/publiccodenet/about/blob/restructure-activities/codebasestewardship/activities/codebase-stewardship/index.md)

### CODE REVIEW

**Review by ericherman — APPROVED** [review]
Ah, good.

File: activities/codebase-stewardship/index.md [hunk 1]
```diff
@@ -35,7 +104,6 @@ We can provide codebase stewardship from the beginning of the development or for
 
 * [codebase in Odoo](odoo-codebase-template.md)
 * [stewardship proposal we can send to principal maintainers](stewardship-proposal-template.md)
-* [Standard for Public Code auditing template](../codebase-auditing/review-template.md)
```
Comment (ericherman) [inline activities/codebase-stewardship/index.md:38]: I see this is removed; is it linked from elsewhere?

Comment (clausmullie) [reply activities/codebase-stewardship/index.md:38]: Auditing is linked several places on the page, but might indeed be good to have the template here again. I've readded it, and we can deduplicate links later.

**Comment by clausmullie** [comment]
Further ideas:
- [ ] remove the success-of-a-codebase page, as we have never used this
- [ ] remove existing-projects link in first paragraph, as it comes later as resource

**Review by ericherman — APPROVED** [review]

**Review by ericherman — APPROVED** [review]
If this passes tests, I think we can merge it as-is and do additional work in another PR.

**Review by Ainali — APPROVED** [review]

**Pull request merged**